src/ansys/fluent/core/launcher/slurm_launcher.py

```diff
@@ -281,6 +281,7 @@ def __init__(
         py: Optional[bool] = None,
         gpu: Optional[bool] = None,
         cwd: Optional[str] = None,
+        custom_fluent_path: Optional[str] = None,
```

Comment:
@prmukherj Is this argument appropriate here?

Reply:
@mkundu1, could you please confirm if this is required for Slurm launcher?

Reply:
@prmukherj we could go ahead and merge meanwhile.